server/controllers/ArticleController.js

```diff
@@ -50,14 +50,19 @@ class ArticleController {
   }
 
   /**
-   * controller to create an article
-   * @param {object} req
-   * @param {object} res
-   * @returns {void}
+   * @description controller to fetch all articles
+   * @param {object} req - request object
+   * @param {object} res - response object
+   * @returns {object} - returns all articles
    */
   static fetchAllArticles(req, res) {
+    const { page } = req.params;
+    const limit = 10;
+    const offset = (+page - 1) * limit;
     Article
       .findAndCountAll({
+        offset,
+        limit,
```

Comment:
Have you handled search?

Reply:
@mbilesanmi no I have not

Reply:
When would this be done?

Reply:
@mbilesanmi its a separate task on the pt-board. So when this pagination task is approved I will start on that task